### ISSUE TITLE
Handle unsecaped > in attribute

### ISSUE THREAD

**Issue by richardmward** [nikku/easysax#17]
I have come across an issue with this using bpmn-js. My xml is generated in java, and is then imported into bpmn-js. Given the following snippet:

```
<startEvent id="event_0ni68v5" name="Start>"/>
```

I get the following error:
```
Error: "closing tag mismatch"
	error webpack-internal:///./node_modules/saxen/dist/index.esm.js:70:10
	handleError webpack-internal:///./node_modules/saxen/dist/index.esm.js:190:13
	parse webpack-internal:///./node_modules/saxen/dist/index.esm.js:892:18
	parse webpack-internal:///./node_modules/saxen/dist/index.esm.js:300:5
	fromXML webpack-internal:///./node_modules/moddle-xml/lib/read.js:844:7
```

I believe attributes should be allowed to have `>` unescaped (https://www.w3.org/TR/xml/#NT-AttValue).

**Comment by nikku** [comment]
Could you provide me with a failing test case that reproduces the given behavior?

**Comment by LukasHechenberger** [comment]
Hey, I ran into the same issue, did you make any progress on this?

Failing test case: https://runkit.com/lukashechenberger/saxen-17

**Comment by LukasHechenberger** [comment]
BTW: This error only occurs with self-closing elements.

**Comment by nikku** [comment]
No progress was made. 

Thanks for sharing your test cases. Could you transform that into a PR? Cf. [existing tests](https://github.com/nikku/saxen/blob/master/test/elements.js).

**Comment by LukasHechenberger** [comment]
There you go

**Comment by nikku** [comment]
Thanks. This is a good starting point for fixing the issue. 

I cannot promise any immediate progress on this one though. The current logic postpones attribute parsing and assumes that `>` is always the element tag closer. A fix means a larger rewrite of the libraries internal workings.

**Comment by nikku** [comment]
Closed via https://github.com/nikku/saxen/pull/20.

Thanks to @matsumotosyu for the small and smart fix :clap:.

**Comment by nikku** [comment]
Fixed via `v8.1.1`.
